src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
 // write code here
+
+const list = document.querySelector('ul');
+const items = [...list.children];
+
+function sortList(elements) {
+  [...elements]
+    .sort((a, b) => {
+      const salaryA = formatedSalary(a.getAttribute('data-salary'));
+      const salaryB = formatedSalary(b.getAttribute('data-salary'));
```

Comment:
The function name `formatedSalary` seems to have a typo. It should be `formattedSalary` to correctly reflect its purpose.